.gitignore

```diff
@@ -20,6 +20,7 @@ dist
 builds/
 disc/
 builds.zip
+docs
```

Comment:
Only ignore `docs/jsdocs` please! Other docs are good, just trying to reduce things that can be rebuilt on each commit.